.pylintdict

```diff
@@ -389,3 +394,23 @@ zsh
 zz
 äquivalenzverbot
 über
+cysteine
+methionine
+phenylalanine
+isoleucine
+leucine
+valine
+tryptophan
+tyrosine
+alanine
+glycine
+threonine
+serine
+asparagine
+glutamine
+aspartic
+glutamic
+histidine
+arginine
+lysine
+proline
```

Comment:
Can you please sort the entire file such that these appear in the alphabetically correct locations? I find that easier to manage and keep updated.